### ISSUE TITLE
AArch64 easyinstall.sh Script Issue

### ISSUE THREAD

**Issue by mcfadyeni** [cuberite/cuberite#5221]
Client version: N/A
Server OS: Linux - Ubuntu 20.04 64-bit AArch64 - Raspberry Pi 4
Cuberite Commit ID: N/A

### Expected behavior
Cuberite downloads and installs as is normal on other ARM64 or AMD64 machines.

### Actual behavior
```
Identifying kernel: Linux
Identifying platform: aarch64
Downloading precompiled binaries.
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now`
```
If manually installed via downloading the tar file directly, it works as normal. However, since the install script only looks for ARM* and not AArch64 (which the Pi4 is being detected as when running Ubuntu Server 64-bit), it does not export the proper URL and further steps fail.

### Steps to reproduce the behavior
Run any machine that uses the 'AArch64' architecture, verified with output from `uname -a`. Download and execute the easyinstall.sh script at `https://raw.githubusercontent.com/cuberite/cuberite/master/easyinstall.sh`

Install script fails with error.

While Cuberite can be downloaded directly from the tar files, this is an issue when running via Docker or K8s, which uses the easyinstall.sh script.

**Comment by tigerw** [comment]
So we have no arm64 builds atm but as you say arm32 binaries can run. Easiest may be to patch the script to redirect 64 to the 32 binaries. In the future we can look into adding buildserver support for arm64.

**Comment by mcfadyeni** [comment]
Alright. Thanks!

**Comment by bearbin** [comment]
Reopening, this issue is valid and has not yet been fixed, although the fix will be very easy as @tigerw mentioned.

**Comment by tigerw** [comment]
Sorry, I should've made it clear it was something for _us_ to do when commenting!

**Comment by mcfadyeni** [comment]
From my testing, I've found that the ARM version works fine on AArch64. Is this a simple matter of adding a case in the $PLATFORM switch inside the script? If it is, I can create a Pull Request.

**Comment by mcfadyeni** [comment]
Fixed in  #5237.
